appveyor.yml

```diff
@@ -25,6 +24,17 @@ test:
     - '**\Sarif.Driver.UnitTests.dll'
     - '**\Sarif.FunctionalTests.dll'
     - '**\Sarif.ValidationTests.dll'
+    
+artifacts:
+  - path: 'bld\bin'
```

Comment:
Let's just skip it. We public the NuGet packages to nuget.org, and I've never needed to get the Nuget package file for a version that we _haven't_ published to NuGet.org.